vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -493,7 +493,9 @@ bool FrameworkElementViewManager::UpdateProperty(
             states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Expanded)] =
                 !innerValue.IsNull() && innerValue.AsBoolean();
             states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Collapsed)] =
-                innerValue.IsNull() || !innerValue.AsBoolean();
+                innerValue.IsNull() ? false : !innerValue.AsBoolean();
```

Comment:
Does this fix the behavior in the bug? if `innerName == "expanded"` that means expanded is present in the object, so collapsed should be set to false if expanded is true? Is someone sets `expanded: null` that's not really a supported scenario.

Reply:
If `expanded: null`, then wouldn't the `innerValue.isNull()` evaluate to true, which would mean that collapsed would be false? My logic for the change was that if `expanded: null`, then it's not been set as a property, which means it shouldn't be expanded or collapsed, which in that case both expanded and collapsed would be set to false.

Reply:
Yeah, but the API is: expanded is either present with true/false or completely absent, not ever supposed to be set to null.

If your logic fixes the narrator reporting that's good, but there's a greater issue that UIA shouldn't probably see that the element could be expanded/collapsed pattern **at all** if expanded isn't set. Which can be a separate bug.